backend/services/mgmt/v1alpha1/connection-data-service/connection-data.go

```diff
@@ -30,14 +32,21 @@ import (
 	sqlmanager_shared "github.com/nucleuscloud/neosync/backend/pkg/sqlmanager/shared"
 	connectionmanager "github.com/nucleuscloud/neosync/internal/connection-manager"
 	neosync_dynamodb "github.com/nucleuscloud/neosync/internal/dynamodb"
+	neosyncgob "github.com/nucleuscloud/neosync/internal/gob"
 	myutil "github.com/nucleuscloud/neosync/internal/mysql"
 	pgutil "github.com/nucleuscloud/neosync/internal/postgres"
 	querybuilder "github.com/nucleuscloud/neosync/worker/pkg/query-builder"
+
 	"go.mongodb.org/mongo-driver/bson"
 	"golang.org/x/sync/errgroup"
 	"google.golang.org/protobuf/types/known/structpb"
 )
 
+var _ = func() any { //nolint:unparam
```

Comment:
this is strange.
how is this different than using `func init()` ?

Reply:
no this should use init. updated it